### DISCUSSION TITLE
How to queue parallel plots

### DISCUSSION THREAD

**Discussion by Xefier** [ericaltendorf/plotman#282] [Q&A] [answered]
How do I configure it so it will create 5 plots in parallel at the exact same time, and then once those plots are complete do 5 plots in parallel again and so on without specifying any specific times. 

Also is plotman aware of plots being done by the Chia GUI? Like could you tell it to wait for those plots to finish and then do 5 in parallel.

**Comment by BasilHorowt** [comment, marked as answer]
>How do I configure it so it will create 5 plots in parallel at the exact same time, and then once those plots are complete do 5 plots in parallel again and so on without specifying any specific times.

Generally it is not advisable to start plots at the exact same time, so set a global stagger of 5 minues, then 5 plots will start over a period of 5 minutes

>Also is plotman aware of plots being done by the Chia GUI?

It should be, but there were some bugs which should be resolved in latest release
